scholly/base/src/global/data.js

```diff
@@ -20,6 +22,8 @@ let TEAM_INFO;
 let TEAM_MEMBERS;
 let TEAM_HOMEPAGE;
 let WEB_PAGES;
+let TEAM_AWARDS;
+let LAYOUT_OPTION;
```

Comment:
You also need to set `LAYOUT_OPTION` for prod
```
LAYOUT_OPTION = env.REACT_APP_LAYOUT_OPTION
    ? JSON.parse(env.REACT_APP_LAYOUT_OPTION)
    : <default>;
```

Reply:
This issue will be resolved on a new PR by @kdav108